include/ignition/physics/ConstructEmpty.hh

```diff
@@ -45,13 +50,18 @@ class ConstructEmptyWorldFeature : public virtual Feature
 };
 
 /////////////////////////////////////////////////
+/// \brief This feature constructs an empty Model entity and return its pointer
+/// from the index-specified World.
```

Comment:
I'm not sure what you mean by "index-specified". I would say "from the given World."